overlord/ifacestate/helpers.go

```diff
@@ -859,6 +859,32 @@ func getHotplugAttrs(task *state.Task) (ifaceName, hotplugKey string, err error)
 	return ifaceName, hotplugKey, err
 }
 
+func obtainHotplugSeq(st *state.State) (int, error) {
+	var seq int
+	if err := st.Get("hotplug-seq", &seq); err != nil && err != state.ErrNoState {
+		return 0, err
+	}
+	seq++
+	st.Set("hotplug-seq", seq)
+	return seq, nil
+}
+
+func isHotplugChange(chg *state.Change) bool {
+	return strings.HasPrefix(chg.Kind(), "hotplug-")
+}
+
+func getHotplugChangeAttrs(chg *state.Change) (int, string, error) {
```

Comment:
This could use named return values, eg. 
```
func getHotplugChangeAttrs(chg *state.Change) (seq int, key string, err error) {
	if err = chg.Get("hotplug-key", &key); err != nil {
		return 0, "", fmt.Errorf("internal error: hotplug-key not set on change %q", chg.Kind())
	}
	if err = chg.Get("hotplug-seq", &seq); err != nil {
		return 0, "", fmt.Errorf("internal error: hotplug-seq not set on change %q", chg.Kind())
	}
	return seq, key, nil
}
```

Reply:
Done.